Makefile

```diff
@@ -16,6 +16,11 @@ clean:
 iso_creation: init
 	handle_user_data='$(HANDLE_USER_DATA)' cert_gen='$(CERT_GEN)' envsubst < $(KICKSTART_TEMPLATE) > $(BUILD_DIR)/$(KICKSTART_FILE)
 	cd $(BUILD_DIR); sudo livecd-creator --config $(BUILD_DIR)/$(KICKSTART_FILE) --logfile=$(BUILD_DIR)/livecd-creator.log --fslabel $(ISO_NAME)
+	# http://askubuntu.com/questions/153833/why-cant-i-mount-the-ubuntu-12-04-installer-isos-in-mac-os-x
+	# http://www.syslinux.org/wiki/index.php?title=Doc/isolinux#HYBRID_CD-ROM.2FHARD_DISK_MODE
+	dd if=/dev/zero bs=2k count=1 of=/tmp/tmp.iso
+	dd if=$(BUILD_DIR)/$(ISO_NAME) bs=2k skip=1 >> /tmp/tmp.iso
```

Comment:
Looking good. Only comment, I'd not write to /tmp. I am a proponent of doing everything in the _build_ directory. This guarantees that once you delete the _build_ directory "you have not more skeletons in the closet"